jdbc/src/main/java/camp/nextstep/jdbc/core/SqlQueryFunction.java

```diff
@@ -0,0 +1,7 @@
+package camp.nextstep.jdbc.core;
+
+import java.sql.SQLException;
+
+public interface SqlQueryFunction<T, R> {
```

Comment:
```suggestion
@FunctionalInterface
public interface SqlQueryFunction<T, R> {
```

명시적으로 FunctionalInterface 를 붙여 주는 것도 좋을 것 같습니다!